src/ripple/rpc/handlers/ServerInfo.cpp

```diff
@@ -22,13 +22,297 @@
 #include <ripple/app/reporting/P2pProxy.h>
 #include <ripple/json/json_value.h>
 #include <ripple/net/RPCErr.h>
+#include <ripple/protocol/LedgerFormats.h>
+#include <ripple/protocol/SField.h>
+#include <ripple/protocol/TER.h>
+#include <ripple/protocol/TxFormats.h>
+#include <ripple/protocol/digest.h>
 #include <ripple/protocol/jss.h>
 #include <ripple/rpc/Context.h>
 #include <ripple/rpc/Role.h>
-#include <ripple/rpc/impl/TransactionSign.h>
+
+#include <boost/algorithm/string.hpp>
+
+#include <unordered_map>
 
 namespace ripple {
 
+namespace detail {
+
+class ServerDefinitions
+{
+private:
+    std::string const
```

Comment:
Remove the `const` from the return type. I suspect this is just a typo from when you changed the param to `const&`

Reply:
Fixed in https://github.com/XRPLF/rippled/pull/4703/commits/b628b4203dd54345ce73597005be6a3e1727be7f